src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1791,22 +1790,11 @@ static void logEx(final String... lines) {
 		for (final String line : lines)
 			SkriptLogger.LOGGER.severe(EXCEPTION_PREFIX + line);
 	}
-	
-	public static String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
-	
-//	static {
-//		Language.addListener(new LanguageChangeListener() {
-//			@Override
-//			public void onLanguageChange() {
-//				final String s = Language.get_("skript.prefix");
-//				if (s != null)
-//					SKRIPT_PREFIX = Utils.replaceEnglishChatStyles(s) + ChatColor.RESET + " ";
-//			}
-//		});
-//	}
-	
+
+	public static Message SKRIPT_PREFIX = new Message("skript.prefix");
```

Comment:
I don't *think* this needs to be public but up for debate

Reply:
I believe I only did public as we had the old one marked as such. Would any other part in skript ever call for this message? If not then yes I can swap to private

Reply:
Not sure, you can probably check pretty easily :)